src/smartdc/bin/pkgsrc-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# CDDL HEADER START
+#
```

Comment:
Oh hell missed this earlier.  Use CDDL 1.0, not 1.1.  See piadm.sh for an example.